### PR TITLE
Add method to get numa nodes with memory

### PR DESCRIPTION
**Pull request by sathnaga**
Add method to get numa nodes with memory

Signed-off-by: Satheesh Rajendran <sathnaga@linux.vnet.ibm.com>

### CODE REVIEW

File: virttest/utils_misc.py [hunk 1]
```diff
@@ -1732,6 +1733,23 @@ def read_from_node_meminfo(self, node_id, key):
         """
         return self.get_all_node_meminfo()[node_id][key]
 
+    def get_online_nodes_withmem(self):
```
Comment (balamuruhans) [inline virttest/utils_misc.py]: I see another PR working on this same API https://github.com/avocado-framework/avocado-vt/pull/1134

**Comment by sathnaga** [comment]
@balamuruhans Looks like @hsmj1412 has closed his PR in-order to get this implementation, can we merge if no further comments, so that the dependency PR can be re-worked/merged, Thanks in advance.

**Comment by balamuruhans** [comment]
Thanks @sathnaga. lets merge it.

**Comment by sathnaga** [comment]
Thanks @balamuruhans

**Pull request merged**